components/shared/net/lib.rs

```diff
@@ -567,18 +568,18 @@ impl ResourceFetchTiming {
         ResourceFetchTiming {
             timing_type: timing_type,
             timing_check_passed: true,
-            domain_lookup_start: 0,
+            domain_lookup_start: Duration::ZERO,
             redirect_count: 0,
-            secure_connection_start: 0,
-            request_start: 0,
-            response_start: 0,
-            fetch_start: 0,
-            redirect_start: 0,
-            redirect_end: 0,
+            secure_connection_start: Duration::ZERO,
+            request_start: Duration::ZERO,
+            response_start: Duration::ZERO,
+            fetch_start: Duration::ZERO,
+            redirect_start: Duration::ZERO,
+            redirect_end: Duration::ZERO,
             connect_start: 0,
             connect_end: 0,
-            response_end: 0,
-            start_time: 0,
+            response_end: Duration::ZERO,
+            start_time: Duration::ZERO,
```

Comment:
Instant might be a better type for these as these a theoretically times.

Reply:
I also thought so, should it be `Instant ` or `SystemTime ` ?

Reply:
From the documentation it seems that `Instant` is guaranteed to be monotonic, but `SystemTime` is not. Since we are subtracting in some places to find a `Duration`, I guess `Instant` might be a better fit. That said, I'm not sure if it's possible to properly create the initial value for `Instant`. It might require  bit of experimentation. Perhaps the unix epoch is a good initial value or using an `Option` or `OnceCell`.